checkstyle/import-control-share-coordinator.xml

```diff
@@ -30,11 +30,18 @@
     <!-- anyone can use public classes -->
     <subpackage name="coordinator">
         <subpackage name="share">
+            <allow pkg="org.apache.kafka.common" />
```

Comment:
It might be better to use the way other import controls are structured generally with 
```
<!-- anyone can use public classes -->
  <allow pkg="org.apache.kafka.common" exact-match="true" />
  ...
```

```
<!-- protocol, records and request/response utilities -->
  <allow pkg="org.apache.kafka.clients" />
  <allow pkg="org.apache.kafka.common.message" />
  <allow pkg="org.apache.kafka.common.protocol" />